cognee/tasks/sentiment_analysis/sentiment_analysis.py

```
@@ -0,0 +1,81 @@
+            "score": round(polarity, 3),
+            "context": combined_text,  # Note: Be careful with PII in production!
+        }
+        logger.debug(
+            f"Sentiment analysis completed: user_id={user_id}, sentiment={sentiment}, score={round(polarity,3)}"
+        )
+        return result
+    except Exception as e:
+        logger.error(
+            f"Sentiment analysis failed: user_id={user_id}, error={e}", exc_info=True
+        )
+        raise ValueError("Sentiment analysis processing failed") from e
```

Ruff (0.14.4)

Avoid specifying long messages outside the exception class

(TRY003)